fft2d/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-add_executable(fft2dRajaSequential.exe fft2dRajaSequential.cxx)
-add_executable(fft2dRajaOmp.exe fft2dRajaOmp.cxx)
+add_executable(fft2d.exe fft2d.cxx fft2dRajaSequential.cxx fft2dRajaOmp.cxx fft2dSequential.cxx)
+add_dependencies(fft2d.exe CALIPER)
+target_link_libraries(fft2d.exe RAJA CALIPER ${RT_LIBRARIES})
 
-target_link_libraries(fft2dRajaSequential.exe RAJA ${RT_LIBRARIES})
```

Comment:
I've changed it so that they all run from the same executable and just linked to the other files